query/src/main/scala/filodb/query/exec/MetadataExecPlan.scala

```diff
@@ -120,8 +120,8 @@ final case class PartKeysExec(queryContext: QueryContext,
     }
     Kamon.currentSpan().mark("creating-resultschema")
     val sch = new ResultSchema(Seq(ColumnInfo("TimeSeries", ColumnType.BinaryRecordColumn),
-      ColumnInfo("StartTime", ColumnType.LongColumn),
-      ColumnInfo("EndTime", ColumnType.LongColumn)), 3,
+      ColumnInfo("_startTime_", ColumnType.LongColumn),
```

Comment:
Will it be even clear if it is `_partStartTime_` ?

Reply:
Lets leave it short. I dont want the long name. It is for debugging anyway

Reply:
renamed actually